andela/ah-orcas
# #161254656 Implement  Email Verification

#### What does this PR do?
```This PR implements email/account verification feature ```
#### Description of Task to be completed?
```On successful registration, an email should be sent to a user with a link to verify the account ```
#### How should this be manually tested?
```Clone this repository: $ https://github.com/andela/ah-orcas.git```
```navigate into the repository directory after cloning```
```$ git checkout ft-email-verification-161254658 ```
```$ python manage.py runserver then launch postman```
``` Register user on postman and then check your email address provided ```
#### Any background context you want to provide?
```Successful registration with valid email provides verification link```
#### What are the relevant pivotal tracker stories?
[#161254658](https://www.pivotaltracker.com/n/projects/2205267/stories/161254658) 
#### Screenshots 
![image](https://user-images.githubusercontent.com/25586421/47706751-1c40f380-dc3b-11e8-979a-a46897a98330.png)
